pkg/apis/core/validation/validation.go

```diff
@@ -6781,7 +6781,8 @@ func ValidateLimitRange(limitRange *core.LimitRange) field.ErrorList {
 			min[string(k)] = q
 		}
 
-		if limit.Type == core.LimitTypePod {
+		// We allow default and defaultRequest when Pod Level Resources are enabled
+		if limit.Type == core.LimitTypePod && !utilfeature.DefaultFeatureGate.Enabled(features.PodLevelResources) {
```

Comment:
This needs to accommodate rollback: if the defaults are already in use, they should be allowed. To do this, add a field to the PodValidationOptions. See https://github.com/kubernetes/kubernetes/blob/fe27448ee4aa82a9cf108974d8e0892a26877c1b/pkg/api/pod/util.go#L375C6-L375C44

Reply:
@tallclair do you mean the default values applied for the pod? If there's pod with default values applied then this logic: https://github.com/kubernetes/kubernetes/blob/fe27448ee4aa82a9cf108974d8e0892a26877c1b/pkg/api/pod/util.go#L1085 takes care of not dropping the pod-level requests and limits. 

Is there a way to preserve defaults in the limit range object as well?